### ISSUE TITLE
error "ClassName does not match" when creating PVC for existing PV using the Rancher 2.1.8 UI (and 2.6.9)

### ISSUE THREAD

**Issue by 3pns** [rancher/rancher#39519]

**Comment by slickwarren** [comment]
Transferring to team 1: 
The issue is that the standard user, cluster owner does not have access to list storage classes nor PVs
<img width="606" alt="Screen Shot 2022-11-07 at 3 36 46 PM" src="https://user-images.githubusercontent.com/16691014/200438213-2b2317fa-a8f0-4b20-96f9-33690902c4d6.png">
<img width="608" alt="Screen Shot 2022-11-07 at 3 36 39 PM" src="https://user-images.githubusercontent.com/16691014/200438217-b561bb84-d2b7-4c35-8c7c-61cdef7460c3.png">

**Comment by MbolotSuse** [comment]
@mantis-toboggan-md 

I took a look at this from the backend side using a setup provided by @slickwarren - I think there's a frontend bug in seeing if the user has access to these resources, and was wondering if you could check my logic. 

I suspect that the message visibility is controlled based on the schema - namely, if the user has the required verbs on the persistentvolume schema. From what I can see, the user doesn't have that schema in the local cluster (i.e. I don't see anything in `$RANCHER/v1/schemas/persistentvolume` ), but it is in the schema for the downstream cluster (i.e. I see verbs as expected in `$RANCHER/k8s/clusters/$CLUSTER/v1/schemas/persistentvolume`). This leads me to believe that the UI is performing the permission check in the wrong place - they need to use the remote cluster's schema/steve (you can see persistent volumes by going to `$RANCHER/k8s/clusters/$CLUSTER/v1/persistentvolume` in your browser).

In short - this looks to me like the UI is checking permissions through the wrong endpoint. Would you mind clarifying what endpoint the UI is using to determine if the user does/does not have access to persistent volume/ storage class resources?
